OracleTuxedo/core/dockerfiles/12.1.3/init.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Please rewrite the following two `echo` lines to:

This sample Docker image is provided by Oracle and is designed to be used as a base image from which you can build Tuxedo-based applications. 

To help you get started customizing this image, we've provided two sample applications ("shm" and "mp") in the Oracle Docker Images GitHub repository. These samples demonstrate how to deploy applications, source code, configuration files and the other necessary settings to build Tuxedo-enabled application containers.

For more information on this image, please visit the Oracle Docker Images GitHub repository at https://github.com/oracle/docker-images.

For more information on Tuxedo, please read the Oracle Tuxedo documentation at http://docs.oracle.com/cd/E53645_01/tuxedo/docs12cr2/index.html.